src/kanata/mod.rs

```diff
@@ -1236,6 +1250,18 @@ impl Kanata {
                             }
                             pbtn
                         }
+                        CustomAction::MoveMouseSpeed { .. } => {
+                            log::debug!("resetting move_mouse_speed");
+                            // Revert the intervals to their old values
+                            if let Some(mmsh) = &mut self.move_mouse_state_horizontal {
+                                mmsh.interval = mmsh.interval * self.move_mouse_speed / 100;
```

Comment:
This calculation isn't always correct for some number where a result of integer division gets truncated e.g. `5 / 2 * 2` will equal `4`, not `5`.

__________

It can even cause panic in some configs, for example holding `(mwheel-left 5 1 1 10)` and pressing `(movemouse-speed 33)` 5 times will cause this panic:
```
thread '<unnamed>' panicked at 'attempt to subtract with overflow', src/kanata/mod.rs:583:41
```
In this example each subsequent press decreases interval: 4, 3, 2, 1, 0 and panics because u16 can't be -1